### DISCUSSION TITLE
Is it possible to migrate a team discussion to a repo discussion?

### DISCUSSION THREAD

**Discussion by blerner** [community/community#2859] [Discussions]
We were excited to try out discussions among teams earlier when they were introduced, but honestly we could never remember where to find them... Having discussions per repo seems great, and we're looking forward to trying it out.  Is it possible to migrate team discussions (preferably with their comments-so-far) to a repo, similarly to how we can migrate issues (with their comments) from one repo to another?

**Comment by becca** [comment]
Right now there is no way to migrate Team Discussions to the new [Repo] Discussions. If there were a migration path how would you define which team posts go to which repository?

**Reply by blerner** [reply]
Unless I'm misunderstanding, I would imagine something similar to the current "Transfer discussion" that appears on this discussion itself:
![image](https://user-images.githubusercontent.com/918464/103237587-052ee680-4916-11eb-95f7-ef1278052e40.png)
![image](https://user-images.githubusercontent.com/918464/103237594-0cee8b00-4916-11eb-9911-c7ad01416e53.png)
Since I'm suggesting moving _Team_ discussions, I would imagine you'd offer "Choose another repository `<that team>` owns to move this discussion to:".
